certification/internal/policy/container/has_modified_files.go

```diff
@@ -76,12 +79,15 @@ func (p *HasModifiedFilesCheck) getDataToValidate(ctx context.Context, imgRef ce
 			}
 			// also add it to the overall list of files.
 			files = append(files, layerFiles)
+			wg.Done()
```

Comment:
Im not sure if this is coding style or practicality, but should this be at the top? ie
```
go func(){
defer wg.Done()
...
}
```
the above seems more common, but as I said I'm not sure the reason. We also have a for loop in this code, that references the channel, so maybe that's why you have it at the end?

Reply:
I’m good either way. I like using defer for things that are very close to their context. As in, opening a file and deferring the close, which happens usually within a couple of lines.

This is small enough to me, so I could go either way. That said, knowing exactly that the Done call happens right after appending the files read clearly to me.

Let me know if you want it changed.

Reply:
I just wanted to make sure it worked the same way since this is inside of a go routine, and since it does I'm fine with what is here. It's probably more readable this way, since it's after the channel call.